pages/index.tsx

```diff
@@ -1,83 +1,88 @@
-        />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <ConnectButton />
-
-        <h1 className={styles.title}>
-          Welcome to <a href="">RainbowKit</a> + <a href="">wagmi</a> +{' '}
-          <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.tsx</code>
-        </p>
-
-        <div className={styles.grid}>
-          <a href="https://rainbowkit.com" className={styles.card}>
-            <h2>RainbowKit Documentation &rarr;</h2>
-            <p>Learn how to customize your wallet connection flow.</p>
-          </a>
-
-          <a href="https://wagmi.sh" className={styles.card}>
-            <h2>wagmi Documentation &rarr;</h2>
-            <p>Learn how to interact with Ethereum.</p>
-          </a>
-
-          <a
-            href="https://github.com/rainbow-me/rainbowkit/tree/main/examples"
-            className={styles.card}
-          >
-            <h2>RainbowKit Examples &rarr;</h2>
-            <p>Discover boilerplate example RainbowKit projects.</p>
-          </a>
-
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h2>Next.js Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
-          >
-            <h2>Next.js Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
-        </div>
-      </main>
-
-      <footer className={styles.footer}>
-        <a href="https://rainbow.me" target="_blank" rel="noopener noreferrer">
-          Made with ❤️ by your frens at 🌈
-        </a>
-      </footer>
-    </div>
+    <Flex direction="column" align="center" justify="center" pt="2rem">
+      <Heading as="h1" size="4xl" mb="4rem">
+        Shield
+      </Heading>
+      <Box width="24rem">
+        <FormControl>
+          <FormLabel>Recipient address</FormLabel>
+          <Input
+            type="string"
+            variant="outline"
+            size="lg"
+            placeholder="vitalik.eth"
+            pr="4.5rem"
+            height="4rem"
+            mb=".75rem"
+          />
+        </FormControl>
+        <FormControl>
+          <FormLabel>Token</FormLabel>
+          <Select placeholder="Ether" size="lg" height="4rem" mb=".75rem">
```

Comment:
not merge blocking, just a comment: there are some additions we may want to make here, for example including the token address next to the token name, perhaps including the token icon as well. We may have to reply on a different component depending on how chakra does `<Select>`

Reply:
+1 Chose a select for simplicity, but we will need much more functionality. I figured we can handle that in a separate ticket/pr